### ISSUE TITLE
Xiaomi Redmi Note 4 missing the remote app

### ISSUE THREAD

**Issue by h2559** [0x192/Universal-Android-Debloater#569]
Xiaomi Redmi Note 4 (mido)

com.duokan.phone.remotecontroller

this app showed up when i reset my phone, i did a search on UAD and did not see it listed. 
asking can it be added to the list and the ETA? thank you

![IMG_20230209_113311](https://user-images.githubusercontent.com/20052727/217937896-b95bc269-3457-41ab-8f5e-97350bbd995c.jpg)
![IMG_20230209_113305](https://user-images.githubusercontent.com/20052727/217937888-88910d5b-afa6-4e99-a5dc-163956a724db.jpg)
![IMG_20230209_113257](https://user-images.githubusercontent.com/20052727/217937871-b3912d91-d887-48e1-bb5a-17653150c5e9.jpg)

**Comment by AnonymousWP** [comment]
> Xiaomi Redmi Note 4 (mido)
> 
> com.duokan.phone.remotecontroller
> 
> this app showed up when i reset my phone, i did a search on UAD and did not see it listed. asking can it be added to the list and the ETA? thank you

I can add it, but then why did you not follow [the template](https://github.com/0x192/universal-android-debloater/issues/new?assignees=&labels=package%3A%3Aaddition&template=add-new-package-s-.md&title=)?

**Comment by h2559** [comment]
1) in the past i submitted a thread and i dont think the template was there, so i just did what i did before. 

2) also the template looks like code, and i am not a programmer, it was hard for me to decipher that stuff, so i just fell back to 1), if you write the template in normal english so laypeople can understand it at a glance that would be extremely helpful for people like me and i bet you will get more submissions since the barrier to entry is lower.

**Comment by AnonymousWP** [comment]
> 1. in the past i submitted a thread and i dont think the template was there, so i just did what i did before.
> 2. also the template looks like code, and i am not a programmer, it was hard for me to decipher that stuff, so i just fell back to 1), if you write the template in normal english so laypeople can understand it at a glance that would be extremely helpful for people like me and i bet you will get more submissions since the barrier to entry is lower.

1. Not a valid reason to me, it shows laziness and not wanting to put effort into it. But I can understand if the template looks confusing.
2. It's not 'code', it's Markdown and is in no way related that you need to be a 'programmer'. Maybe the following is easier for you to understand? Hopefully you can answer the following if I put it like this:

    - Did you remove the packages on your phone? Yes or no?
    - Package name: 
    - Removal: 
    - Description of the package: 
    - Screenshot:

See https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen for more information.

**Comment by h2559** [comment]
definitely, plain english is always best, you can't assume who is willing to sign up for a github account just to post a suggestion (like me, that is exactly what i did)

if you can add one last point, ask to attach any screenshots, if you find that helpful, cause a picture is worth a thousand words.

**Comment by AnonymousWP** [comment]
> definitely, plain english is always best, you can't assume who is willing to sign up for a github account just to post a suggestion (like me, that is exactly what i did)
> 
> if you can add one last point, ask to attach any screenshots, if you find that helpful, cause a picture is worth a thousand words.

Updated. Can you fill it in then?

**Comment by AnonymousWP** [comment]
> > > definitely, plain english is always best, you can't assume who is willing to sign up for a github account just to post a suggestion (like me, that is exactly what i did)
> > > if you can add one last point, ask to attach any screenshots, if you find that helpful, cause a picture is worth a thousand words.
> > 
> > 
> > Updated. Can you fill it in then?
> 
> No, of course not, who do you think you are, on the internet and in real life? I already submitted the requested info based on the crappy template in the first post, you have all the info that you need, if you want to fill in another template then do it yourself.

I'm just trying to help by preparing a simple template for you so that you can fill it in, and I can submit a PR. Why so offended and having to make it personal? That's so low. Where did you put it then?

I see it now, you created new issues. Next time, just tell me in a normal way instead of having a big mouth. That can save a lot unnecessary hassle. :) And next time just create one issue for your package, not a separate package per issue.

**Comment by h2559** [comment]
> > > > definitely, plain english is always best, you can't assume who is willing to sign up for a github account just to post a suggestion (like me, that is exactly what i did)
> > > > if you can add one last point, ask to attach any screenshots, if you find that helpful, cause a picture is worth a thousand words.
> > > 
> > > 
> > > Updated. Can you fill it in then?
> > 
> > 
> > No, of course not, who do you think you are, on the internet and in real life? I already submitted the requested info based on the crappy template in the first post, you have all the info that you need, if you want to fill in another template then do it yourself.
> 
> I'm just trying to help by preparing a simple template for you so that you can fill it in, and I can submit a PR. Why so offended and having to make it personal? That's so low. Where did you put it then?
> 
> I see it now, you created new issues. Next time, just tell me in a normal way instead of having a big mouth. That can save a lot unnecessary hassle. :) And next time just create one issue for your package, not a separate package per issue.

Hey asshole, next time don't try to give orders over the internet and then try to play it off when it didn't go your way.

Your original template is garbage, it was stupid of you to create it, don't try to force people to use it, you are a programmer, don't think you are all that you bitch.

Like I said earlier, all the info you need is in the very first post, go add the app to the list now for removal so everyone can benefit, Github isn't the place for a powertripping nerd like you to act all petty about a shitty template.

**Comment by AnonymousWP** [comment]
> I already deleted the app, for those who were not helpful you can go fuck yourselves.

Great. Hereby you have been blocked. :)

**Comment by Obegg** [comment]
@h2559 you should be grateful.
Why?
Because this project is dead, no one else was going to reply to you.
There's an active fork of this project and the absolute chad over here @AnonymousWP actually is helping keep the fork active with his PRs.
The only word I can find is ungrateful.
Anyway, @AnonymousWP thank you.

**Comment by Obegg** [comment]
> If this project died it is because of users like

Can you stop? Please.
This project has owner and he abandoned this project, the one who keeps it alive is the one you are disrespecting.

@AnonymousWP Ignore him, thank you for all you are doing, love you.
